extensions/blocks/podcast-player/view.js

```diff
@@ -0,0 +1,55 @@
+/* global _wpmejsSettings, MediaElementPlayer, jetpackPodcastPlayers */
+/**
+ * Internal dependencies
+ */
+import './style.scss';
+
+const playerInstances = {};
+const meJsSettings = typeof _wpmejsSettings !== undefined ? _wpmejsSettings : {};
+
+const initializeBlock = function( id ) {
+	const block = document.getElementById( id );
+	if ( block ) {
+		const player = {
+			id,
+			block,
+			audio: block.querySelector( 'audio' ),
+			currentTrack: 0,
+		};
+		playerInstances[ id ] = player;
+
+		// Initialize player UI.
+		player.mediaElement = new MediaElementPlayer( player.audio, meJsSettings );
+	}
+};
+
+// Initialze queued players.
+if ( window.jetpackPodcastPlayers !== undefined ) {
+	jetpackPodcastPlayers.forEach( initializeBlock );
+}
+
+// Replace the queue with an immediate initialization for async loaded players.
+window.jetpackPodcastPlayers = {
+	push: initializeBlock,
+};
+
+// Add global handler for clicks.
+window.addEventListener( 'click', function( e ) {
+	// Prevent handling clicks if a modifier is in use.
+	if ( e.shiftKey || e.metaKey || e.altKey ) {
+		return;
+	}
+
+	// Check if the clicked element was episode link.
+	const audioUrl = e.target.getAttribute( 'data-podcast-audio' );
```

Comment:
Should we scope the data attr to `jetpack`. That way it's more resilient should someone else decide to code a similar 3rd party block which could _easily_ use a similarly named data attr.

Reply:
like `data-jetpack-podcast-…`?

Reply:
Done that, let me know if it's ok. I'd like to remove it in a subsequent PR and load data from the encoded JSON instead of the data attribute. This just gets us running without too much overhead

Reply:
That's good.